lib/screens/sentence_list_screen.dart

```diff
@@ -61,7 +61,12 @@ class _SentenceListScreenState extends State<SentenceListScreen> {
         itemCount: data.length,
         itemBuilder: (context, index) {
           Sentence item = data[index];
-          return SentenceItem(phrase: item, index: index+1);
+          //return SentenceItem(phrase: item, index: index+1);
```

Comment:
Is this going to be used in the future? I mean SentenceItem(....)

Reply:
No, I don't think so. I deleted the class SentenceItem